From 5cab480f1472af368594c1d275cff507123cb8eb Mon Sep 17 00:00:00 2001
From: Petr Tesarik <ptesarik@suse.cz>
Date: Tue, 28 Jul 2015 17:39:10 +0900
Subject: [PATCH 2/2] [PATCH 2/2] Remove notes about ELF being the only available format for Xen dumps.

Since Xen dumps can now be stored in compressed formats, adjust
documentation that claims the opposite.

Signed-off-by: Petr Tesarik <ptesarik@suse.com>
---
 makedumpfile.8 |    5 +----
 print_info.c   |    5 +++--
 2 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/makedumpfile.8 b/makedumpfile.8
index 2d38cd0..3f66d73 100644
--- a/makedumpfile.8
+++ b/makedumpfile.8
@@ -16,7 +16,7 @@ makedumpfile \- make a small dumpfile of kdump
 .br
 \fBmakedumpfile\fR \-g \fIVMCOREINFO\fR \-x \fIVMLINUX\fR
 .br
-\fBmakedumpfile\fR \-E [\-\-xen-syms \fIXEN-SYMS\fR|\-\-xen-vmcoreinfo \fIVMCOREINFO\fR] \fIVMCORE\fR \fIDUMPFILE\fR
+\fBmakedumpfile\fR    [\fIOPTION\fR] [\-\-xen-syms \fIXEN-SYMS\fR|\-\-xen-vmcoreinfo \fIVMCOREINFO\fR] \fIVMCORE\fR \fIDUMPFILE\fR
 .br
 \fBmakedumpfile\fR \-\-dump-dmesg [\-x \fIVMLINUX\fR|\-i \fIVMCOREINFO\fR] \fIVMCORE\fR \fILOGFILE\fR
 .br
@@ -441,7 +441,6 @@ This option is mainly for debugging.
 \fB\-\-xen-syms\fR \fIXEN-SYMS\fR
 Specify the \fIXEN-SYMS\fR with debug information to analyze the xen's memory usage.
 This option extracts the part of xen and domain-0.
-\-E option must be specified with this option.
 .br
 .B Example:
 .br
@@ -453,7 +452,6 @@ Specify \fIVMCOREINFO\fR instead of \fIXEN-SYMS\fR for analyzing the xen's memor
 .br
 \fIVMCOREINFO\fR should be made beforehand by makedumpfile with \-g option, and
 it contains the xen's information.
-\-E option must be specified with this option.
 .br
 .B Example:
 .br
@@ -464,7 +462,6 @@ it contains the xen's information.
 Exclude all the user domain pages from Xen kdump's \fIVMCORE\fR, and extracts the
 part of xen and domain-0. If \fIVMCORE\fR contains \fIVMCOREINFO\fR for Xen, it is
 not necessary to specify \fI\-\-xen-syms\fR and \fI\-\-xen-vmcoreinfo\fR.
-\-E option must be specified with this option.
 .br
 .B Example:
 .br
diff --git a/print_info.c b/print_info.c
index 9c36bec..2342525 100644
--- a/print_info.c
+++ b/print_info.c
@@ -87,10 +87,11 @@ print_usage(void)
 	MSG("\n");
 	MSG("\n");
 	MSG("  Creating DUMPFILE of Xen:\n");
-	MSG("  # makedumpfile -E [--xen-syms XEN-SYMS|--xen-vmcoreinfo VMCOREINFO] VMCORE DUMPFILE\n");
+	MSG("  # makedumpfile [-c|-l|-p|-E] [--xen-syms XEN-SYMS|--xen-vmcoreinfo VMCOREINFO]\n");
+	MSG("    VMCORE DUMPFILE\n");
 	MSG("\n");
 	MSG("  Filtering domain-0 of Xen:\n");
-	MSG("  # makedumpfile -E -d DL -x vmlinux VMCORE DUMPFILE\n");
+	MSG("  # makedumpfile [-c|-l|-p|-E] -d DL -x vmlinux VMCORE DUMPFILE\n");
 	MSG("\n");
 	MSG("  Generating VMCOREINFO of Xen:\n");
 	MSG("  # makedumpfile -g VMCOREINFO --xen-syms XEN-SYMS\n");
-- 
1.7.1

